request-filterer/request-filterer-impl/src/test/java/com/duckduckgo/request/filterer/impl/plugins/CookiesFeatureTogglesPluginTest.kt

```diff
@@ -0,0 +1,152 @@
+/*
+ * Copyright (c) 2022 DuckDuckGo
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package com.duckduckgo.request.filterer.impl.plugins
+
+import com.duckduckgo.app.CoroutineTestRule
+import com.duckduckgo.appbuildconfig.api.AppBuildConfig
+import com.duckduckgo.request.filterer.api.RequestFiltererFeatureName.RequestFilterer
+import com.duckduckgo.request.filterer.store.RequestFiltererFeatureToggleRepository
+import kotlinx.coroutines.ExperimentalCoroutinesApi
+import kotlinx.coroutines.test.runTest
+import org.junit.Assert.assertEquals
+import org.junit.Assert.assertFalse
+import org.junit.Assert.assertNull
+import org.junit.Assert.assertTrue
+import org.junit.Before
+import org.junit.Rule
+import org.junit.Test
+import org.mockito.kotlin.mock
+import org.mockito.kotlin.whenever
+
+@ExperimentalCoroutinesApi
+class RequestFiltererFeatureTogglesPluginTest {
+
+    @get:Rule var coroutineRule = CoroutineTestRule()
+    lateinit var testee: RequestFiltererFeatureTogglesPlugin
+
+    private val mockFeatureTogglesRepository: RequestFiltererFeatureToggleRepository = mock()
+    private val mockAppBuildConfig: AppBuildConfig = mock()
+
+    @Before
+    fun before() {
+        testee = RequestFiltererFeatureTogglesPlugin(mockFeatureTogglesRepository, mockAppBuildConfig)
+    }
+
+    @Test
+    fun whenIsEnabledAndFeatureIsNotACookieFeatureThenReturnNull() = runTest {
```

Comment:
The function names in this file should be `RequestFiltererFeature` instead of `CookieFeature`

Reply:
Done